src/http/mod.rs

```diff
@@ -28,11 +28,29 @@ pub use server_context::ServerContext;
 
 use self::server_config::ServerConfig;
 
-pub fn max_age(res: &Response) -> Option<Duration> {
+pub fn cache_policy(res: &Response) -> Option<CacheControl> {
   let header = res.headers.get(CACHE_CONTROL)?;
   let value = header.to_str().ok()?;
-  let policy = CacheControl::from_value(value)?;
-  policy.max_age
+
+  CacheControl::from_value(value)
+}
+
+pub fn max_age(res: &Response) -> Option<Duration> {
+  match cache_policy(res) {
+    Some(value) => value.max_age,
+    None => None,
+  }
+}
+
+pub fn cache_visibility(res: &Response) -> String {
```

Comment:
Is this fn used anywhere other than tests?

Reply:
No, @meskill. Just the tests. Would you suggest moving it somewhere else?

Reply:
yeah, you could put such things into the tests module if you need it only there

another thing to consider: do we need this at all? The Cachability struct could be compared directly without need to convert it to string